OpenSim/Common/Storage.cpp

```diff
@@ -2623,7 +2615,7 @@ void Storage::interpolateAt(const Array<double> &targetTimes)
         StateVector vec;
         // INTERPOLATE THE STATES
         ny = getDataAtTime(t,ny,&y);
-        vec.setStates(t,ny,y);
+        vec.setStates(t, SimTK::Vector_<double>(ny, y));
 
```

Comment:
Can you update the documentation so it doesn't refer to removed parameters? Thanks.